samples/system-test/encryption.test.js

```diff
@@ -87,3 +88,19 @@ it('should rotate keys', () => {
   );
   assert.include(output, 'Encryption key rotated successfully.');
 });
+
+it('should convert CSEK to KMS key', async () => {
+  const encryptedFileName = 'encrypted-file';
+  await bucket
+    .file(encryptedFileName, {
+      encryptionKey: key,
+    })
+    .save();
```

Comment:
```suggestion
    .save('test);
```
I believe `data` is a required param for `file#save()`

Reply:
Done !!